svf/lib/SABER/SaberSVFGBuilder.cpp

```diff
@@ -99,17 +100,32 @@ void SaberSVFGBuilder::collectGlobals(BVDataPTAImpl* pta)
     }
 }
 
-PointsTo& SaberSVFGBuilder::CollectPtsChain(BVDataPTAImpl* pta,NodeID id, NodeToPTSSMap& cachedPtsMap)
+PointsTo& SaberSVFGBuilder::CollectPtsChain(BVDataPTAImpl* pta, NodeID id, NodeToPTSSMap& cachedPtsMap)
```

Comment:
Could make some comments to explain this design choice based on your issue and also leave the issue link here too (https://github.com/SVF-tools/SVF/issues/991)?

Thanks,

Reply:
Ok.